packages/frontend/src/page/components/answer-form/answer/index.tsx

```diff
@@ -38,10 +38,28 @@ export const Answer = ({
     const purelyNumerical = isAnswerPurelyNumerical(question);
     const invalid = isAnswerInvalid(question);
     const answeredTooSoon = isAnsweredTooSoon(question);
+    const pendingArbitration = question.pendingArbitration;
 
     return (
         <div className="flex flex-col justify-between gap-3">
```

Comment:
I think to make checks a bit better we can update the various `isAnswer` functions etc all across the codebase to include the `pendingArbitration` check no?

Reply:
I think it would be better to have a dedicated `isAnswerPendingArbitration` because we have cases where we only want to check if there's any arbitration in progress; also initializing an arbitration causes the question to go in the `finalized` state, and since we need to display different things when a question is `finalized but under arbitration` or `finalized and not arbitration`, we need the checks to be separated.